dali/sasha/sacmd.cpp

```diff
@@ -110,6 +111,7 @@ class CSashaCommand: public CInterface, implements ISashaCommand
         mb.read(state);
         mb.read(owner);
         mb.read(cluster);
+        mb.read(filterScopes);
```

Comment:
this will break backward compatibility..
You can see [horrible] code below which is looking for newer field values on the end of the buffer .. it would have go there if anywhere. I hate how inflexible it is.

It's tempting to change the protocol (e.g. to a json or serialized IPT), but this [server] would have to cope with the current legacy format, and newer clients would have to check if server could cope with new format somehow, e.g. maybe in accept if message is blank, send back version #.

This should be a separate self-contained task though, and prob. OTT for this, but you will need to ensure backward compat. with this change.